### ISSUE TITLE
Changing the size of a type does not change the size of the type inheriting from it

### ISSUE THREAD

**Issue by xusheng6** [Vector35/binaryninja-api#4607]
Thanks @emesare for creating these steps!

- Create empty view
- Add the following types:
```c
struct baseStruct __packed {}; // Make sure this is zero-sized

struct __base(baseStruct, 2) test __packed
{
    uint8_t first;
    uint8_t second;
};
```
- Add `uint8_t` field to `baseStruct`
- `bv.types["test"].width` incorrectly reports 2 instead of 3

**Comment by xusheng6** [comment]
Related to https://github.com/Vector35/binaryninja-api/issues/4604

Related to #4542

**Comment by D0ntPanic** [comment]
Fixed in 3.6.4750
